tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -28,16 +28,12 @@ public function testDiffReturnsDiff()
         $diff = <<<'TXT'
 --- Original
 +++ New
-@@ @@
+@@ -1,3 +1,4 @@
  <?php
  class Foo extends Bar {
 -    function __construct($foo, $bar) {
 +    public function __construct($foo, $bar)
 +    {
-         $this->foo = $foo;
```

Comment:
I hope not ;)
I changed the output builder to not add same lines before and after a diff chunk, within a chunk this OK. 
Side note;
The amount of not change lines between a different lines before creating a new chunk is not configurable within `diff 2`, it will be in a new output builder I'm writing. I could an option to display "context" lines before and after each chunk if there is a need (or when I find the time).